fetch2/src/main/java/com/tonyodev/fetch2/DownloadManager.java

```diff
@@ -54,6 +54,30 @@ void pause(long id) {
         actionProcessor.processNext();
     }
 
+    void pause(final String groupId) {
```

Comment:
What if the download hasn't started yet? Can you pause a download that has been queued? 

For example if you queue 10 downloads and have a limit of 3 concurrent downloads, then try to pause them all. 

Do only the 3 running downloads get paused, and then the next 7 start immediately? Or are all 10 paused?